projects/origin-release/golang-1.10/Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN yum install -y epel-release && \
     git config --system user.email origin-release@redhat.com
 
 ADD *.rpm /tmp/origin-rpm/
-RUN yum install -y /tmp/origin-rpm/*.rpm && \
+RUN yum install -y /tmp/origin-rpm/*.rpm libvirt-devel && \
```

Comment:
Hm, is there a reason why you haven't added the package to the big `INSTALL_PKGS` list ten lines above this?